vagrant/README.md

```diff
@@ -4,7 +4,7 @@
 
 Please install the following:
 
-- [VirtualBox](https://www.virtualbox.org/)
+- [VirtualBox](https://www.oracle.com/virtualization/technologies/vm/downloads/virtualbox-downloads.html)
```

Comment:
1. https://www.oracle.com/virtualization/technologies/vm/downloads/virtualbox-downloads.html points to `downloads.virtualbox.org`

1. `virtualbox.org` seems to be better choice when it works (most of the time)